### PR TITLE
Alert and Cutoff unnecessary timer

### LINKED ISSUE #145: display error/warning messages on bad requests FE
Should use SessionController

### LINKED ISSUE #180: Specify loading time & remove for short operations
- a note on the modal to say, for example, "this can take 30-60 seconds" 
- remove or reduce timer for short operations like editing trip name

### PR DESCRIPTION
**Pull request by NLmeng**
closes #180 closes #145

### CODE REVIEW

File: client/src/components/common/AlertSnackbar.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export function AlertSnackbar({
+  open,
+  handleClose,
+  message,
+  severity = 'error',
+}) {
+  return (
+    <Snackbar
+      anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
```
Comment (Graywing13) [inline client/src/components/common/AlertSnackbar.js:19]:
```suggestion
      anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
```

Comment (Graywing13) [reply client/src/components/common/AlertSnackbar.js:19]: Well actually idk... it's a lot less noticeable at the bottom so maybe top is better?

Comment (NLmeng) [reply client/src/components/common/AlertSnackbar.js:19]: i prefer top

**Review by Graywing13 — APPROVED** [review]
lgtm

**Pull request merged**